### PR TITLE
search bar added in countries and universites

### PR DESCRIPTION
**Pull request by sumedh-mohod**
search bar functionality added in countries and universities

### CODE REVIEW

File: app/Http/Controllers/CountryController.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\User;
```
Comment (ithqindia) [inline app/Http/Controllers/CountryController.php:5]: Remove this

File: app/Http/Controllers/UniversityController.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 namespace App\Http\Controllers;
 
+use App\Models\User;
```
Comment (ithqindia) [inline app/Http/Controllers/UniversityController.php:5]: Remove this

**Review by ithqindia — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/UniversityController.php (hunk 2 above)
Comment (sumedh-mohod) [reply app/Http/Controllers/UniversityController.php]: done

**Comment by sumedh-mohod** [comment]
ckeditor added but not working

**Review by ithqindia — APPROVED** [review]

**Comment by ithqindia** [comment]
Merged locally in https://github.com/ithqindia/metroclub/pull/97

**Pull request closed without merging**